docs/detections/visual-event-analyzer.asciidoc

```diff
@@ -0,0 +1,93 @@
+[[visual-event-analyzer]]
+[role="xpack"]
+== Visual event analyzer 
+
+Elastic Security allows any event detected by Elastic Endpoint to be analyzed using a process-based visual analyzer. This enables security analysts to drill down into all related hosts, processes, and other events to aid in their investigations. 
+
+[float]
+[[find-events-analyze]]
+=== Find events to analyze
+
+You can only visualize events triggered by hosts configured with the Elastic Endpoint Security Integration. In KQL, this translates to any event with the `agent.type` set to `endpoint`.
+
+To access events that can be visually analyzed:
+
+1. In Elastic Security, select **Hosts** > **Events**. A list of all your host's events appears in the bottom panel of the host's view.
+
+2. Create a KQL query that filters all `endpoint` detected events by entering `agent.type: endpoint and process.entity_id : *` into the KQL search bar, and then select **Update**.
++
+[role="screenshot"]
+image::images/kql-agent-type.png[]
++
+A list of all `endpoint` related events appears.
+
+3. Select the **Analyze Event** icon next to the event you want to analyze. 
++
+[role="screenshot"]
+image::images/analyze-event.png[]
+
+The visual **Analyzer** view appears.
+
+[role="screenshot"]
+image::images/analyze-event-view.png[]
+
+You can also analyze events from <<timelines-ui,Timelines>>
+
+
+[discrete]
+[[visual-analyzer-ui]]
+=== Visual event analyzer UI
+
+Inside the visual analyzer, each cube represents either a node, process, or alert. Click and drag in timeline view to see all process relationships. 
+
+To make the analyzer full screen, select the **Full Screen** icon above the left-panel. 
+
+[role="screenshot"]
+image::images/full-screen-analyzer.png[]
+
+The left panel contains a list of all processes related to the event, starting with the event chain's first process. **Analyzed Events** are highlighted by a light blue outline around the cube.
+
+[role="screenshot"]
+image::images/process-list.png[]
+
+In the graphical view, you can:
+
+- Zoom in and out of the graphical view using the slider to the right of the timeline.
+- Click and drag around the graphical view to more process relationships.
+- See the time passed between each node.
+- See all events related to each process. 
```

Comment:
At this time, all nodes are processes and vice versa. This list uses `node` and `process` interchangeably but it might be more clear to stick to just one term.